src/material/chips/chips.md

```diff
@@ -1 +1,194 @@
-<!-- TODO: this guide and the related live examples need to be rewritten to reflect the new chips API. -->
+Chips allow users to view and interact with a list of information.
+
+### Static Chips
+
+Chips are always rendered in a container. To create chips, start with a `<mat-chip-set>` element. Then, nest `<mat-chip>` elements inside the `<mat-chip-set>`.
+
+<!-- example(chips-overview) -->
+
+By default, `<mat-chip>` renders a chip with Material Design styles applied. For a chip with no styles applied, use `<mat-basic-chip>`.
+
+<em>Hint: `<mat-basic-chip>` receives the `mat-mdc-basic-chip` CSS class in addition to the `mat-mdc-chip` class. </em>
+
+#### Adding Icons
+
+A chip has two spots to display icons using content projection.
```

Comment:
Might be worth mentioning here that the icons are optional.

Reply:
I re-wrote part about icons. LMK what you think